.gitignore

```diff
@@ -5,3 +5,5 @@ lib
 /npm-packages-offline-cache
 tmp
 yarn
+/package-lock.json
```

Comment:
Yeah that's what we're doing, but npm5 (node 8) generates that package-lock, which is why it felt helpful to add it here.